### DISCUSSION TITLE
Minimalist Chatbot UI

### DISCUSSION THREAD

**Discussion by toi500** [FlowiseAI/Flowise#2109] [Show and tell]
## Minimalist Chatbot UI for Flowise

<img width="1322" alt="Screenshot 2024-04-05 144151" src="https://i.ibb.co/ncR98YT/Screenshot-2024-04-05-181743.png">

**Live Demo:**

[http://chatbot.flowise.surge.sh](http://chatbot.flowise.surge.sh)

**GitHub Repository:**

[https://github.com/toi500/chatbot_ui_flowise](https://github.com/toi500/chatbot_ui_flowise)

**Note:**

While this integration works with Flowise, it lacks functionalities like chat history, uploading pictures, starter prompts, etc.

**Discussion:**

Flowise would benefit from having more native options for UIs. The current design is good, but it is also difficult to customize if you don't know what you're doing. Unfortunately, I only have very basic JS knowledge and can't help with that.

**Comment by amansoni7477030** [comment]
Hi @toi500, please check https://github.com/FlowiseAI/FlowiseChatEmbed/pull/183 . Wait for it to be approved. Now it will be very responsive on any platform. I made changes in many files and also added the close button at the top of the chat window. Now it will work very responsively.

**Comment by toi500** [comment]
@amansoni7477030 thank you so much for doing this. I truly appreciate it.

**Reply by amansoni7477030** [reply]
Your're welcome!

**Comment by ricardogonzalezcases** [comment]
Are there any solution to update chatflowconfigs after init? I need update additional context fron my app in Next requests... Thank you
